### ISSUE TITLE
Look into double facebook topic, maybe bug

### ISSUE THREAD

**Issue by danblah** [ResetNetwork/recoding-tech#16]

**Comment by danblah** [comment]
This is a blocker for assigning the facebook "topic" to policy action cause we don't know which facebook to use.

**Comment by danblah** [comment]
Also same for the double united states topic.

**Comment by rudietuesdays** [comment]
Will investigate, likely import script bug

**Comment by danblah** [comment]
Rocking, lemme know when somethings up for testing!

**Comment by jheretic** [comment]
As far as I can tell, the bug that caused this and #15 is no longer occurring. But there are enough entries in Zotero now that it's causing Sanity to timeout on updating them all in one go, so I'm refactoring the import script to batch them into multiple transactions before closing this out.

**Comment by jheretic** [comment]
The changes to the import script above to batch imports and improve performance was merged in https://github.com/ResetNetwork/recoding-tech/pull/37. The reason for these two duplicates sticking around was due to them being referenced by an old draft in the datastore dating from before policies and policy_actions were renamed (so it didn't show up in the interface). I was able to remove that old draft, and then delete the offending duplicates. They did not recur on an import run.
